packages/api/src/platforms/vtex/resolvers/offer.ts

```diff
@@ -78,7 +79,18 @@ export const StoreOffer: Record<string, Resolver<Root>> = {
     }
 
     if (isOrderFormItem(root)) {
-      return root.sellingPrice / 1e2
+      return root.sellingPrice / 1e2 / root.unitMultiplier
```

Comment:
This is a modification in the Packing feature. I discovered during this task that the sellingPrice is the only price that is already multiplied by the unitMultiplier. Because of this, some inconsistencies are happening, like a spotPrice being greater than the listPrice due to the multiplication.

This modification should only affect stores that utilize the unitMultiplier, which, at this time, is not being used by any stores.

Reply:
If the store doesn't have a `unitMultiplier`, what will happen here? Do we have a default value set?

Reply:
The unit multiplier should default to a value of 1. To ensure greater reliability, I will add a redundancy measure.